installer/templates/phx_web/components/core_components.ex

```diff
@@ -64,51 +64,39 @@ defmodule <%= @web_namespace %>.CoreComponents do
         @kind == :info && "alert-info",
         @kind == :error && "alert-error"
       ]}>
-        <.icon :if={@kind == :info} name="hero-information-circle-mini" class="size-5 shrink-0" />
-        <.icon :if={@kind == :error} name="hero-exclamation-circle-mini" class="size-5 shrink-0" />
+        <.icon :if={@kind == :info} name="hero-information-circle-mini" class="h-5 w-5 shrink-0" />
+        <.icon :if={@kind == :error} name="hero-exclamation-circle-mini" class="h-5 w-5 shrink-0" />
```

Comment:
Shouldn't we keep these as size-5?

Reply:
Ok, looking below, it seems you got the wrong core components file? :D

Reply:
Seems like they weren’t in sync for me :O